src/qibolab/__init__.py

```diff
@@ -75,3 +78,46 @@ def execute_qasm(circuit: str, platform, initial_state=None, nshots=1000):
     return QibolabBackend(platform).execute_circuit(
         circuit, initial_state=initial_state, nshots=nshots
     )
+
+
+def get_available_platforms() -> list[str]:
+    """Returns the platforms found in the $QIBOLAB_PLATFORMS directory."""
+    return [
+        d.name
+        for d in get_platforms_path().iterdir()
+        if d.is_dir()
+        and Path(f"{os.environ.get(PLATFORMS)}/{d.name}/platform.py") in d.iterdir()
+    ]
+
+
+class MetaBackend:
```

Comment:
Following a private discussion with @BrunoLiegiBastonLiegi, I think the `MetaBackend` class may not be needed as a whole, but only the `load` method. However, this is strictly dependent on https://github.com/qiboteam/qibo/pull/1282 and other repos, so depending on what is decided there, feel free to ignore this comment.

Reply:
Yes, the only required bit is the `load` part, definitely.
However, the `list_available` could also be convenient, that's why the idea was to collect them in a `MetaBackend`. But two separate functions would also work as well (and, if we really want to be minimal, even just `load` could be enough... but I still consider a nice feature to be able to query the possible backends in the present environment).